workflow-bot-app/src/test/java/com/symphony/bdk/workflow/engine/camunda/monitoring/repository/ActivityCmdaApiQueryRepositoryTest.java

```diff
@@ -0,0 +1,111 @@
+package com.symphony.bdk.workflow.engine.camunda.monitoring.repository;
+
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.ArgumentMatchers.contains;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.symphony.bdk.workflow.converter.ObjectConverter;
+import com.symphony.bdk.workflow.monitoring.repository.domain.ActivityInstanceDomain;
+
+import org.camunda.bpm.engine.HistoryService;
+import org.camunda.bpm.engine.history.HistoricActivityInstance;
+import org.camunda.bpm.engine.history.HistoricVariableInstance;
+import org.camunda.bpm.engine.history.NativeHistoricVariableInstanceQuery;
+import org.camunda.community.mockito.QueryMocks;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Collections;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ExtendWith(MockitoExtension.class)
+class ActivityCmdaApiQueryRepositoryTest {
```

Comment:
We can maybe test that the `findAllByWorkflowInstanceId` filters activities and only returns serviceTask ones..

Reply:
i may not understand this comment. The first test actually is testing the scenario returning serviceTask only.